pkg/apis/configuration/validation/policy.go

```diff
@@ -243,6 +243,9 @@ func validateOIDC(oidc *v1.OIDC, fieldPath *field.Path) field.ErrorList {
 	if oidc.AuthEndpoint == "" {
 		return field.ErrorList{field.Required(fieldPath.Child("authEndpoint"), "")}
 	}
+	if oidc.LogoutEndpoint == "" {
```

Comment:
I am not sure if `logoutEndpoint` and `redirectPostLogout` should be required fields. This will lead to a breaking change for current users looking to upgrade to the new release, as they will need to modify their OIDC configurations.